src/main/webapp/assets/modules/contest-detail/contestDetailDirective.js

```diff
@@ -120,12 +145,21 @@ techlooper
       }
     };
   })
- .directive('acceptance', function () {
-  return {
-    restrict: "E",
-    replace: true,
-    templateUrl: "modules/contest-detail/contestDetailAcceptance.html",
-    link: function (scope, element, attr, ctrl) {
-    }
-  };
-});
\ No newline at end of file
+  .directive('acceptance', function () {
+    return {
+      restrict: "E",
+      replace: true,
+      templateUrl: "modules/contest-detail/contestDetailAcceptance.html",
```

Comment:
'ctrl' is defined but never used.<br>'attr' is defined but never used.<br>'element' is defined but never used.<br>'scope' is defined but never used.